### PR TITLE
feat: Improvments as per #2

### PR DESCRIPTION
**Pull request by merrcury**
@yakkomajuri Made the changes as per #2

### CODE REVIEW

File: __tests__/index.js [hunk 1]
```diff
@@ -2,43 +2,40 @@ const { createEvent } = require('@posthog/plugin-scaffold/test/utils.js')
...
+                accountSID: 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
+                authToken: 'your_auth_token',
+                senderPhoneNumber: '+15555555555',
+                triggeringEventsAndNumber: 'event1:+15555555555,event2:+15555555555',
+                timeout: -1,
+            },
+            global: {},
+        })
+        // Fail test if above expression doesn't throw anything.
+      expect(true).toBe(false);
```
Comment (yakkomajuri) [inline __tests__/index.js]: this is not a great pattern.

Jest has utilities to check if a function throws.

File: __tests__/index.js [hunk 2]
```diff
@@ -2,43 +2,40 @@ const { createEvent } = require('@posthog/plugin-scaffold/test/utils.js')
...
+                authToken: 'your_auth_token',
+                senderPhoneNumber: '+15555555555',
+                triggeringEventsAndNumber: 'event1:+15555555555,event2:+15555555555',
+                timeout: 366 * 60 * 60 * 24 + 1,
+            },
+            global: {},
+            
+        })
+        // Fail test if above expression doesn't throw anything.
+      expect(true).toBe(false);
```
Comment (yakkomajuri) [inline __tests__/index.js]: same as above

File: index.js [hunk 3]
```diff
@@ -13,26 +11,20 @@ async function setupPlugin({ config, global }) {
     }
 
     global.senderPhoneNumber = config.senderPhoneNumber
-    if (config.timeout <= 0 || config.timeout > 365 * 60 * 60 * 24) {
-        console.error(`timeout is not supported`)
-        return 'timeout is not supported'
+    const maxTimeout = 365 * 24 * 60 * 60
```
Comment (yakkomajuri) [inline index.js]:
```suggestion
    const maxTimeout = 365 * 24 * 60 * 60 // one year
```

File: index.js [hunk 4]
```diff
@@ -72,15 +58,14 @@ const jobs = {
...
         return null
     } else {
         const request = {
             eventName: event.event,
         }
-        response = await jobs.sendMessageWithTwilio(request).runNow()
     }
+    const response = await jobs.sendMessageWithTwilio(request).runNow()
 
     return response
```
Comment (yakkomajuri) [inline index.js]: we shouldn't return anything, and this response is useless.
```suggestion
await jobs.sendMessageWithTwilio(request).runNow()
```

**Review by yakkomajuri — COMMENTED** [review]
looking much better, thanks! a few comments

**Comment by merrcury** [comment]
@yakkomajuri I have made the asked changes.

**Review by yakkomajuri — APPROVED** [review]

**Pull request merged**